compiler/damlc/daml-ide-core/BUILD.bazel

```diff
@@ -6,9 +6,16 @@ load(
     "da_haskell_library",
 )
 
+filegroup(
```

Comment:
Given that it’s now a single file, I wouldn’t bother with a `filegroup`.

Reply:
Raising a new PR with this one.